<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>271</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Configurassões</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Cor</string>
     </property>
     <widget class="QSlider" name="corR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="corG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="corB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Dimenssões do cubo</string>
     </property>
     <widget class="QSlider" name="xC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="zC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="yC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Raios do Elipsóide</string>
     </property>
     <widget class="QSlider" name="rX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="rZ">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="rY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Raio da Esfera</string>
     </property>
     <widget class="QSlider" name="raio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>61</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLCDNumber" name="lcdR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>61</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLCDNumber" name="Dx">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Dy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Dz">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>350</y>
       <width>61</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLCDNumber" name="lcdRy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRz">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdRx">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLCDNumber" name="raio_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Elementos</string>
    </property>
    <widget class="QPushButton" name="Cubo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cubo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="noEsfera">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>noEsfera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Esfera">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Esfera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="noCubo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>noCubo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Elipsoide">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Elipsoide</string>
     </property>
    </widget>
    <widget class="QPushButton" name="noElipsoide">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>noElipsoide</string>
     </property>
    </widget>
   </widget>
   <widget class="Folha" name="tela" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>431</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Folha</class>
   <extends>QWidget</extends>
   <header location="global">folha.h</header>
   <container>1</container>
   <slots>
    <slot>recebeArt(Sculptor*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MainWindow</sender>
   <signal>ligaArtista(Sculptor*)</signal>
   <receiver>tela</receiver>
   <slot>recebeArt(Sculptor*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>ligaArtista(Sculptor*)</signal>
 </slots>
</ui>
